content/en/docs/concepts/security/pod-security-admission.md

```diff
@@ -25,7 +25,7 @@ created.
 
 ### Built-in Pod Security admission enforcement
 
-This page is part of the documentation for Kubernetes v{{< skew currentVersion >}}.
+This page is part of the documentation for Kubernetes v{{< skew currentPatchVersion >}}.
```

Comment:
The original is fine; this statement applies to all patch versions in a minor release.